go.sum

```diff
@@ -190,15 +193,20 @@ github.com/magiconair/properties v1.8.0/go.mod h1:PppfXfuXeibc/6YijjN8zIbojt8czP
 github.com/magiconair/properties v1.8.1/go.mod h1:PppfXfuXeibc/6YijjN8zIbojt8czPbwD3XqdrwzmxQ=
 github.com/mailru/easyjson v0.0.0-20180730094502-03f2033d19d5/go.mod h1:C1wdFJiN94OJF2b5HbByQZoLdCWB1Yqtg26g4irojpc=
 github.com/mailru/easyjson v0.7.0/go.mod h1:KAzv3t3aY1NaHWoQz1+4F1ccyAH66Jk7yos7ldAVICs=
+github.com/manland/mattermost-plugin-gitlab v0.3.0 h1:4o5wOvLvdbez918JQz0ZMOTOLA3QDy4BEv2cFHWMeDk=
```

Comment:
Does `github.com/manland/mattermost-plugin-gitlab` still need to be imported?

Reply:
I expect this can be removed too.